pkg/jobs/jobspb/jobs.proto

```diff
@@ -106,8 +106,9 @@ message ImportDetails {
     string name = 18;
     int64 seq_val = 19;
     bool is_new = 20;
-    repeated string target_cols = 21;
```

Comment:
hm, this is actually a migration problem -- pause on 19.2, resume on 20.1 and cols change?